lib/src/connector/ischool_plus_connector.dart

```diff
@@ -36,79 +36,67 @@ class ISchoolPlusConnector {
 
   /// The Authorization Step of ISchool (2023-10-21)
   /// 1. GET https://app.ntut.edu.tw/ssoIndex.do
-  /// 2. POST https://app.ntut.edu.tw/oauth2Server.do (It should be. See the comment on step 2)
-  /// 3. GET https://istudy.ntut.edu.tw/login2.php (It should be. See the comment on step 3)
-  /// 4. do something...
-  static Future<ISchoolPlusConnectorStatus> login(String account) async {
-    String result;
+  /// 2_1. POST https://app.ntut.edu.tw/oauth2Server.do (It should be. See the comment on step 2)
+  /// 2_2. follow the redirection to https://istudy.ntut.edu.tw/login2.php (It should be. See the comment on step 2)
+  static Future<ISchoolPlusConnectorStatus> login(String account, doFirebaseLogin) async {
```

Comment:
Consider making `doFirebaseLogin` an optional named parameter for better flexibility and clarity:

```suggestion
  static Future<ISchoolPlusConnectorStatus> login(String account, {bool logEventToFirebase = true}) async {
```

Reply:
[68c30d3](https://github.com/NEO-TAT/tat_flutter/pull/225/commits/68c30d32b99f5214419b700479bc45297d83533f)